buildSrc/src/main/groovy/io.micronaut.build.internal.test-resources-mntest-extension.gradle

```diff
@@ -1,6 +1,7 @@
 plugins {
     id 'io.micronaut.build.internal.test-resources-simple-module'
     id 'io.micronaut.build.internal.test-fixtures'
+    id "io.micronaut.build.internal.kotlin-base"
```

Comment:
:x: This doesn't look right: all extensions are not Kotlin projects. The Kotlin plugin should only be applied where Kotlin is used.